scss/_base_hr.scss

```diff
@@ -3,16 +3,28 @@
 // Horizontal rule
 @mixin vf-b-hr {
   hr {
-    border: 0;
-    height: 1px;
+    @extend %hr;
+
     margin-bottom: calc(#{$spv-inner--scaleable} - 1px);
-    margin-top: 0;
-    position: relative;
-    width: 100%;
 
     & + p {
       margin-top: -$sp-unit;
     }
+
+    // stylelint-disable-next-line selector-no-qualifying-type
+    &.is-dense {
+      margin-bottom: -1px;
```

Comment:
Adding negative bottom margin seems a bit error prone. Are you sure it's gonna work as expected regardless of what is put after `hr`?

Reply:
Also, is this expected that dense variant has no space below the hr line, but has standard spacing above it?

<img width="630" alt="Screenshot 2021-02-16 at 13 53 46" src="https://user-images.githubusercontent.com/83575/108065561-7e1d0580-705e-11eb-839c-c40eb3f25c45.png">

Reply:
the space above comes from the p that precedes it, space underneath intentional